UI/Web/src/theme/themes/dark.scss

```diff
@@ -33,11 +33,11 @@
     --primary-color-darker-shade: #338A67;
     --primary-color-darkest-shade: #25624A;
     --error-color: #BD362F;
-    --bs-body-bg: #3f4245; // #121212
+    --bs-body-bg: #000; // #121212
```

Comment:
Robbie we can't have this changed. It breaks everything:
![image](https://github.com/user-attachments/assets/67d392c0-4d70-45f3-912e-555af8bd61a1)

Reply:
Reverted already